Content.Client/UserInterface/Controls/RadialContainer.cs

```diff
@@ -67,37 +67,77 @@ public RadialContainer()
     {
 
     }
-	
-    protected override void Draw(DrawingHandleScreen handle)
+
+    /// <inheritdoc />
+    protected override Vector2 ArrangeOverride(Vector2 finalSize)
     {
-		
         const float baseRadius = 100f;
```

Comment:
Constants should be defined at the top of the file

Reply:
need to understand what to do with logic that adds padding by incrementing based on child count - maybe baseRadius is a bad idea? or it should be MinRadius?

Reply:
Renamed Radius with InitialRadius, and old Radius property is CalculatedRadius now, xml-doc explains that it depends on InitialRadius and uses visible child count and const RadiusIncrement to be calculated. CalculatedRadius have private setter for safety reasons